ethereum/core/src/main/java/org/hyperledger/besu/ethereum/bonsai/worldview/BonsaiWorldStateUpdateAccumulator.java

```diff
@@ -600,6 +623,8 @@ private void rollAccountChange(
       }
       if (replacementValue == null) {
         if (accountValue.getPrior() == null) {
+          // TODO: should we remove from the parent accumulated change also?  only if it is a
```

Comment:
@matkt I believe this is a "no" in the current implementation and we can delete this comment.  Do you concur?

Reply:
we don't have parent accumulator in the last implementation, we have only parent key value storage . So no